server/lib/field_hub_web/live/project_show_live.html.heex

```diff
@@ -54,11 +54,49 @@
                         <% %{
                             database: %{
                                 doc_count: db_doc_count,
-                                file_size: db_file_size
+                                file_size: db_file_size,
+                                last_n_changes: last_n_changes
                             },
                             files: files
                         } -> %>
-                            <section>Database documents: <%= db_doc_count %></section>
+                            <section>Database documents: <%= db_doc_count %></section>                      
+                            <section>
+                                
+                                <tr>
+                                    <td valign="top" class="column-last-changes-slider">
+                                        <form id="last_changes_form" phx-change="last_changes">                                        
+                                            <%= @nb_changes_to_display %> last changes                                  
+                                        
+                                            <div class="slider-container">                                             
+                                             <input 
+                                                type="range" 
+                                                min="2" 
+                                                max= "50"
+                                                value= {@nb_changes_to_display}
+                                                class="slider"
+                                                name="n-last-changes"
+                                            > 
+                                            </div>                                       
+                                        </form>
+                                    </td>
+                                    <td> 
+                                        <table>
+                                            <%= for change <- last_n_changes do %> 
+                                                <%{type, date_time, user_name} = CouchService.extract_most_recent_change_info(change) %>
+                                                <tr> 
+                                                    <td>
+                                                        '<%= change["doc"]["resource"]["identifier"] %>' 
+                                                        <br />
+                                                        <%= type %> by user: <i><%= user_name %></i>
+                                                    </td>
+                                                    <td class="column-date-time"><%= Calendar.strftime(date_time, "%c") %></td>                    
+                                                </tr>
+                                            <% end %>                                            
```

Comment:
Maybe we could display the datetime vertically aligned.

Reply:
The changes are now show in a table.
The date and time are aligned, and I set the column "Time" with a fixed width that corresponds exactly to the value.